### PR TITLE
Set spanmetricsprocessor call metric IsMonotonic to true

### PR DESCRIPTION
**Pull request by yeya24**
Signed-off-by: yeya24 <yb532204897@gmail.com>

**Description:** <Describe what has changed.>

As mentioned in the issue, set `IsMonotonic` to true.

**Link to tracking Issue:** https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/2829

**Testing:** Tested manually with prometheusremotewrite exporter and it works well.
Screenshot from 2021-03-23 20-34-10

**Documentation:** <Describe the documentation added.>

### CODE REVIEW

File: processor/spanmetricsprocessor/processor.go [hunk 1]
```diff
@@ -246,7 +246,8 @@ func (p *processorImp) collectCallMetrics(ilm *pdata.InstrumentationLibraryMetri
 
 		mCalls := pdata.NewMetric()
 		mCalls.SetDataType(pdata.MetricDataTypeIntSum)
-		mCalls.SetName("calls")
+		mCalls.SetName("calls_total")
```
Comment (albertteoh) [inline processor/spanmetricsprocessor/processor.go:249]: You may need to update the tests to reflect the new name.

File: processor/spanmetricsprocessor/processor.go [hunk 2]
```diff
@@ -246,7 +246,8 @@ func (p *processorImp) collectCallMetrics(ilm *pdata.InstrumentationLibraryMetri
...
+		mCalls.IntSum().SetIsMonotonic(true)
```
Comment (albertteoh) [inline processor/spanmetricsprocessor/processor.go:250]: Please add a test for this. I think it should be in `verifyConsumeMetricsInput`.

File: processor/spanmetricsprocessor/processor_test.go [hunk 3]
```diff
@@ -307,10 +307,11 @@ func verifyConsumeMetricsInput(input pdata.Metrics, t *testing.T) bool {
 	mi := 0
 	// The first 3 metrics are for call counts.
 	for ; mi < 3; mi++ {
-		assert.Equal(t, "calls", m.At(mi).Name())
+		assert.Equal(t, "calls_total", m.At(mi).Name())
 
 		data := m.At(mi).IntSum()
 		assert.Equal(t, pdata.AggregationTemporalityCumulative, data.AggregationTemporality())
+		assert.Equal(t, true, data.IsMonotonic())
```
Comment (albertteoh) [inline processor/spanmetricsprocessor/processor_test.go:314]: I would use `assert.True`.

**Review by albertteoh — APPROVED** [review]
LGTM, thanks @yeya24! Not sure why the load tests are failing as these changes seem quite minimal. Is anyone able to re-run it? or maybe a commit to address my last comment will trigger another run.

File: processor/spanmetricsprocessor/processor_test.go (hunk 3 above)
Comment (yeya24) [reply processor/spanmetricsprocessor/processor_test.go]: Updated.

**Review by tigrannajaryan — APPROVED** [review]

**Pull request merged**